### DISCUSSION TITLE
gamerule showCoordinates true

### DISCUSSION THREAD

**Discussion by rolo2912git** [itzg/docker-minecraft-bedrock-server#207] [Q&A]
Server is running, so kids are happy enough. Now they are complaining about not being able to see the Coordinates. Read up on it and what is required is to set  gamerule showCoordinates true on the console.
To "access" the console apparently in a terminal window the command: docker attach mincraft_bedrock is to be used mincraft_bedrock being the name of the container. This brings one to a blank line. The setting gamerule showCoordinates can be entered. Questions:

1. Is this the correct path?
2. The container seemed to be stopped once the docker attach command is used
3. Is this the correct spot to enter the command gamerule showCoordinates ?
4. If it is correct, is it persistent even after the container is restarted?
5. How does one get out of the console again? Somewhere on the internet for a bash script  ^M was used. Is this correct?
6. Can gamerule showCoordinates be used after a world is already created?

Thank you in behalf of my son you loves to play mindcraft on this bedrock server.

**Comment by itzg** [comment]
Yes, that's the correct way to access the server console and the right kind of command. To detach without stopping the container, use the sequence ctrl-p ctrl-q, which is documented here if you need it later 

https://github.com/itzg/docker-minecraft-bedrock-server#executing-server-commands

Game rules are persistent and are actually necessary to be done after the world is generated.

**Reply by rolo2912git** [reply]
In behalf of my kids I like to thank you for your answer. Will test it out.
